CanvasPlusPlayground/Services/CanvasService.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CanvasService.swift
+//  CanvasPlusPlayground
+//
+//  Created by Abdulaziz Albahar on 9/14/24.
+//
+
+import Foundation
+
+class CanvasService {
```

Comment:
nit: This can also be a `struct`. If you wanna keep it a `class`, maybe have a shared instance and remove the static from the function.

e.g:
```swift
class CanvasService {
   static let shared: CanvasService = .init()

   func fetch(/* ... */)
}
```

and then in other places we can use it like: `CanvasService.shared.fetch`.

Reply:
Yes! The static functionality was just a temporary measure. `struct` is a good idea!